Samples/nuget.config

```diff
@@ -10,6 +10,8 @@
     <packageSources>
     <clear />
         <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+        <add key="Internal" value="https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json" />
+        <add key="Local" value="C:\_Data\Cobalt\Reunion" />
```

Comment:
Seems like this line is causing issues in the pipeline

Reply:
Also, I think we should not be using the internal feed :)